APE24.rst

```diff
@@ -0,0 +1,443 @@
+2. Otherwise, they will gracefully sunset it from the list of maintained/accepted
+   packages.
+
+After sunsetting, if a package becomes active again, it is up to the package
+maintainer to contact Astropy or pyOpenSci in order to have it be actively
+listed once more. An example scenario that might happen is when the sole
+maintainer goes on a hiatus longer than the time frame set in
+`maintainer responsiveness <https://www.pyopensci.org/software-peer-review/our-process/policies.html#maintainer-responsiveness>`_
+and then comes back to a sunsetted package.
+
+.. _Scientific Python: https://scientific-python.org/
+
+Finding reviewers
+-----------------
+
+pyOpenSci currently has a list of reviewers who have signed up for this task using the
+`reviewer sign-up form`_.
+Because they utilize 2 reviewers for each package, they generally try to find
+a reviewer with domain-specific expertise, while the other with or without.
+Sometimes, the second reviewer will focus instead on general usability,
+ease of installation, documentation quality, or packaging infrastructure.
+For every review, they target a diversity of contributors to ensure that they
+have a mix of varying gender, cultural, etc., identities.
+
+If we decide to partner with pyOpenSci, our reviewers would sign up using
+the pyOpenSci form above, specifying:
+
+* astronomy domain expertise, and
+* membership in the Astropy community.
+
+pyOpenSci does not publicly list all the people who signed up
+(i.e, a reviewer is only public during the review process)
+but the `pyOpenSci Editorial Board`_ has access to the list.
+
+pyOpenSci will respect the Astropy-specific criteria for someone to be a
+reviewer for Affiliated request, namely:
+
+* familiarity with the Astropy project,
+* ability to judge whether a package integrates well with the Astropy ecosystem
+  (as per pre-APE 24 guidelines), and
+* having domain expertise in the area of the package (e.g., galaxy evolution).
+
+pyOpenSci and Astropy both ask editors/reviewers disclose any potential conflict of interest (COI)
+prior to agreeing to review a package. In the event where COI occurs, with this partnership,
+we would follow the
+`pyOpenSci COI process <https://www.pyopensci.org/software-peer-review/our-process/policies.html#conflict-of-interest-for-reviews-and-editors>`_.
+pyOpenSci invites the Astropy community to review this language to ensure it meets our needs.
+
+.. _reviewer sign-up form: https://forms.gle/GHfxvmS47nQFDcBM6
+
+Packaging guidelines
+--------------------
+
+pyOpenSci has been developing a `community-driven packaging guide`_
+that covers modern best practices and recommendations for scientific Python packaging.
+This guide has a stringent community review process. Reviewers for this guide
+represent members of PyPA, core Python, Anaconda (conda/conda-forge), and
+core packages for front- and back-end tools (e.g., ``flit``, PDM, ``hatch``/``hatchling``, etc.).
+It is also maintained by `Scientific Python`_.
```

Comment:
hmmm maybe you are referencing a different guide? we maintain that guide at pyOpenSci but henry's scikit-hep guide is on the scientific python website now. that is a different guide with a very different development process

Reply:
I was referencing to https://www.pyopensci.org/python-package-guide/ . I saw some Scientific Python faces under https://github.com/pyopensci/python-package-guide#contributors- . But you are saying that your guide has nothing to do with Scientific Python? Should I remove this text then?

```rst
It is also maintained by `Scientific Python`_.
```

Reply:
Should I also mention https://learn.scientific-python.org/development/guides/packaging-classic/ or is that not in scope? Hope you can advise. Thanks!

Reply:
hey @pllim some of the scientific python project people absolutely contributed to our guide via our community review process!! but our guide on the pyOpenSci website is maintained by the pyOpenSci  community not the scientific Python project. i know it's confusing but they are separate entities.  The other guide https://learn.scientific-python.org/development/guides/packaging-classic/  is excellent but different from ours and largely developed by Henry from the scikit HEP community. so for our review process and the APE around peer review we probably want to  stick to our guide. but i think both guides are useful so if you want to link to the other guide in your documentation around packaging that is wonderful.

i hope that helps. we just wont consider that other guide a part of our peer review process as it isn't a pyos community   entity that we contribute to or maintain.